create_salsa_project.go

```diff
@@ -43,6 +43,12 @@ func execCreateSalsaProject(args []string) {
 		log.Fatalf("Could not create webhook on salsa project: %s\n", err)
 	}
 
+	for _, branch := range []string{"master", "debian/*", "upstream", "pristine-tar"} {
```

Comment:
Could you add upstream/* in preparation for adopting DEP14 please?

Reply:
done!